src/messaging/js/composeui-messaging-abstractions/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@morgan-stanley/composeui-messaging-abstractions",
+  "version": "0.1.0-alpha.9",
+  "description": "TypeScript abstractions library for ComposeUI's messaging",
+  "private": false,
+  "author": "Morgan Stanley",
+  "license": "Apache-2.0",
+  "type": "module",
+    "files": [
+        "dist/**/*"
+    ],
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "npm run build:clean && npm run build:rollup",
+    "build:clean": "rimraf dist",
+    "build:rollup": "rollup -c",
+    "test": "vitest",
+    "test:run": "vitest run",
+    "test:coverage": "vitest run --coverage"
+  },
+  "publishConfig": {
+    "access": "public",
+    "provenance": true
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/morganstanley/ComposeUI.git"
+  },
+  "dependencies": {
+    "rxjs": "^7.8.0"
+  },
+  "devDependencies": {
+    "@types/node": "^20.0.0",
```

Comment:
The types are for node 20, but the project builds with 24, so let's sort this out,